tycho-core/pom.xml

```diff
@@ -67,22 +67,22 @@
 			</plugin>
 
 			<plugin>
-				<groupId>org.apache.felix</groupId>
-				<artifactId>maven-bundle-plugin</artifactId>
+				<groupId>biz.aQute.bnd</groupId>
```

Comment:
@laeubi is this still required? Or can I just remove it too?

Reply:
If the build works without it can be removed... :-)

Reply:
It looks like the generated manifest was not included in the build jars for a while (at least `tycho-core-3.0.0.jar` and the current 4.0.0-SNAPSHOT) only have a trivial, non-OSGi compliant, MANIFEST.MF.
So I'm pretty sure it can be removed, but lets see what the build says now (I first wanted to fix the other failures).